doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -478,7 +489,7 @@ for further explanations.
 
 One can inform the rewrite tactic about morphisms and relations just
 by using the typeclass mechanism to declare them using the :cmd:`Instance` and
-:cmd:`Context` commands. Any object of type Proper (the type of
+:cmd:`Context` commands. Any object of type ``Proper`` (the type of
```

Comment:
I take it `Proper` is a `Type`?  Maybe at some point we can add a list of such predefined Types in the doc.